angularjs-portal-frame/src/main/webapp/js/services.js

```diff
@@ -31,10 +31,60 @@ app.factory('miscService', function($http, $window, $location) {
     $window._gaq.push(['_trackEvent', category, action, label]);
   }
 
+    var portletMatchesSearchTerm = function(portlet, searchTerm, opts) {
```

Comment:
I'm gonna try a quick and dirty fix to this, but most likely this will need to roll into next week.  I suspect it may be a bit tricky to decouple those pieces.

I was wondering why there was this arbitrary split between "home" and "frame".  Makes more sense now.